.circleci/config.yml

```diff
@@ -52,12 +52,12 @@ jobs:
           command: |
               source activate ersilia
               ersilia serve molecular-weight
-              ersilia run -i "CCC" > output_one_0.json
-              ersilia run -i "CCC" -o output_one_1.json
-              ersilia run -i "CCC" -o output_one_3.csv
-              ersilia run -i test/inputs/compound_list.csv > output_list_0.json
-              ersilia run -i test/inputs/compound_list.csv -o output_list_1.json
-              ersilia run -i test/inputs/compound_list.csv -o output_list_2.csv
+              ersilia api -i "CCC" > output_one_0.json
+              ersilia api -i "CCC" -o output_one_1.json
+              ersilia api -i "CCC" -o output_one_3.csv
+              ersilia api -i test/inputs/compound_list.csv > output_list_0.json
```

Comment:
Hi @rohitgupta29 I think the api command was removed from the CLI and if you pulled changes as @DhanshreeA requested you'll notice...

Reply:
Hi @P-Mefut! 
I am checking it and share about it in some time.

Reply:
Hi @P-Mefut ...Can you share a bit more more details on this? 
For my repo, it does not show code from line 58 (ersilia api -i test/inputs/compound_list.csv > output_list_0.json), after pull also.

Reply:
@rohitgupta29 this is because you first need to sync your fork of ersilia from the original ersilia repo, and then pull those changes locally (because your fork is your remote, and not the original ersilia repo)

Reply:
Thanks for the catch @P-Mefut !